api/deploy/kubernetes/cephfs/csi-config-map.go

```diff
@@ -22,6 +22,7 @@ import (
 	"fmt"
 	"text/template"
 
+	"github.com/ceph/ceph-csi/api/deploy/kubernetes"
```

Comment:
Let's move it to this directory ? 
	"github.com/ceph/ceph-csi/api/deploy/config"

@nixpanic Can you please add your opinion?

Reply:
ConfigMaps are Kubernetes specific, these won't work with other CO's. Under the `kubernetes/` directory is correct imho.